tests/full_game.py

```diff
@@ -1,25 +1,43 @@
 #!/usr/bin/env python
 import sys; sys.path.append("..")
+import logging
 from fireplace import cards
 from fireplace.utils import play_full_game
 
 
 def test_full_game():
-	play_full_game()
+	return play_full_game(False)
 
 
 def main():
+	turns = []
+	runtime = []
+
+	logging.disable(logging.CRITICAL)
```

Comment:
Outputting to the console is very expensive, it slows the simulation down by a factor of 8 on my PC.